README.md

```diff
@@ -52,21 +52,45 @@ using [pdfkit](http://pdfkit.org/). There is an example provided in the examples
 
 The Lob.com API also supports HTML strings in leiu of a file of the above type. See below for examples of submitting with HTML strings.
 
-For templates and more information regarding HTML, please see the [Lob documentation](https://lob.com/docs/node#html-fonts). 
+For templates and more information regarding HTML, please see the [Lob documentation](https://lob.com/docs/node#html-fonts).
 
 ##Usage<a name="Usage"></a>
 ```javascript
 var Lob = require('lob')('YOUR API KEY');
 
-// alternative way to initialize
-var lobFactory = require('lob');
-var Lob = new lobFactory('YOUR API KEY');
-
 // change api version
-var Lob = require('lob')('YOUR API KEY');
-Lob.setVersion('api_version');
+var Lob = require('lob')('YOUR API KEY', 'API_VERSION');
+
+// change internal defaults (e.g. host)
+var options = {/* see options below */};
+var Lob = require('lob')('YOUR API KEY', 'API_VERSION', options);
+
+// callback pattern
+Lob.settings.list({ type: 1 }, function(err, body) {
+  if(err) return callback(err);
+  return callback(null, body.data);
+});
 ```
 
+Additionally, every resource method returns a promise, so you don't have to use the regular callback. E.g.
+
+```javascript
+var Lob = require('lob')('YOUR API KEY').promisify();
```

Comment:
There's no need to promisify any more, correct? This should then be removed from the docs.